sshfs.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: doc
+title: SSHFS Workflow Tutorial
+---
+
+## Why SSHFS?
+
+SSHFS is a tool that allows you to mount any directory from a remote machine on your local machine, and consequently use your favorite text editor/IDE/code editing environment and still be able to compile and run your code on the remote machine.
+
```

Comment:
I would put in here exactly what the sshfs does.

- There exists a copy on the machine over there
- There may (or may not) exist a copy on a machine over here
- SSH keeps them in sync automagically!

Advantages
- You don't have to `svn ci` and `svn up` every time
- Like you said, edit on your machine
- Saves network delays.

Not all of this stuff will make sense to them but we want to sell this idea so they don't ping pong with our svn server.